Remove missing OpenCASCADE header file which is not available
in newer versions.
--- library/NekMesh/CADSystem/OCE/OpenCascade.h.orig
+++ library/NekMesh/CADSystem/OCE/OpenCascade.h
@@ -60,7 +60,6 @@
 #include <BRep_Tool.hxx>
 #include <GCPnts_AbscissaPoint.hxx>
 #include <GProp_GProps.hxx>
-#include <GeomAdaptor_HSurface.hxx>
 #include <GeomLProp_CLProps.hxx>
 #include <GeomLProp_SLProps.hxx>
 #include <ShapeAnalysis_Curve.hxx>
